proto/manifest.proto

```diff
@@ -9,6 +9,8 @@ message Manifest {
 }
 
 message Resource {
+    reserved 12;
```

Comment:
I don't _think_ we need to worry about breaking. The usage here is thin on xattrs. However, the types are actually equivalent.